docs/integrations/vscode-extension/README.md

```diff
@@ -0,0 +1,45 @@
+# DevCycle VSCode Extension 
+## Feature Overview
+
+### Current Features
+
+
+1. **View All Feature Flags**: The variable view in the extension displays a list of all variables existing within your code and your project.
+
+2. **See Code Usages**: The variable view also shows you where each of your DevCycle variables resides in your codebase, providing a convenient click-to-navigate feature.
+
+3. **Understand Feature Status**: Hovering over your DevCycle variables in your code brings up a card detailing information about the variable and the current status of the feature across environments.
+
+## Requirements
+
+Before getting started with DevCycle, make sure you meet the following requirements:
+
+- You need a DevCycle account. Sign up for a free account [here](https://devcycle.com/signup) (no credit card required).
```

Comment:
this signup link does not exist

Reply:
i think Vic was going to clean up the docs in the other repo, but i can make a quick PR to fix the link

Reply:
when we release the new extension version it should generate a PR with the updated link from your changes